### DISCUSSION TITLE
User's action log (interface) / Единый лог действий пользователей (интерфейс)

### DISCUSSION THREAD

**Discussion by Duche3d** [PastVu/pastvu#45] [Ideas / Идеи и предложения]
Логи действий модераторов и администраторов, а возможно и всех пользователей.

**Comment by paul-k-pastvu** [comment]
This is closed, but without available log by user(s)?

**Comment by paul-k-pastvu** [comment]
Ну, у нас есть [целая отдельная метка "Logging" про это ](https://github.com/PastVu/pastvu/issues?q=is%3Aopen+is%3Aissue+label%3ALogging). Я всё расписал в #124. Даже с картинками.
